specification/communication/data-plane/Email/preview/2023-01-15-preview/CommunicationServicesEmail.json

```diff
@@ -0,0 +1,384 @@
+              "x-ms-error-code": {
+                "description": "Error code - this will be the same as the code in the error property in the response body.",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "../../../../../common-types/data-plane/v1/types.json#/definitions/ErrorResponse"
+            }
+          },
+          "202": {
+            "description": "The service has accepted the request and will start processing the message later. It will return 'Accepted' immediately and include an 'Operation-Location' header. Client side should further query the operation/message status using the URL specified in 'Operation-Location' header. Once the send operation has succeeded, you can get additional status about email delivery through event grid events (for events reference, please refer to: https://learn.microsoft.com/en-us/azure/event-grid/communication-services-email-events)",
+            "headers": {
+              "Operation-Location": {
+                "description": "Location url of where to poll the status of this operation from.",
+                "type": "string"
+              },
+              "retry-after": {
+                "description": "This header will only be present when the operation status is a non-terminal status. It indicates the minimum amount of time in seconds to wait before polling for operation status again.",
+                "type": "integer",
+                "format": "int32"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/EmailSendResult"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-examples": {
+          "Send Email": {
+            "$ref": "./examples/SendEmail.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "EmailSendResult": {
+      "description": "Status of the long running operation",
+      "required": [
+        "id",
+        "status"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The unique id of the operation. Use a UUID.",
+          "type": "string",
+          "example": "8540c0de-899f-5cce-acb5-3ec493af3800"
+        },
+        "status": {
+          "description": "Status of operation.",
+          "enum": [
+            "NotStarted",
+            "Running",
+            "Succeeded",
+            "Failed",
+            "Canceled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "EmailSendStatus",
+            "modelAsString": true
+          }
+        },
+        "error": {
+          "description": "Error details when status is a non-success terminal state.",
+          "$ref": "../../../../../common-types/data-plane/v1/types.json#/definitions/ErrorDetail"
+        }
+      }
```

Comment:
Not sure if this is already discussed, would service return the result of the action in a `result` property? @mikekistler

Reply:
Hi @weidongxu-microsoft , We don't have any result for the POST action. We only have a status, which is already returned in the `Status` property of the status monitor.